docs/UserGuide.md

```diff
@@ -434,7 +434,22 @@ Format: `exit`
 **A**: Install the app in the other computer and overwrite the empty data file it creates with the file that contains the data of your previous AddressBook home folder.
 
 --------------------------------------------------------------------------------------------------------------------
+## Glossary
+
+* **Academic level**: Mainstream academic years from primary to tertiary education in Singapore (i.e. Primary 1-6, Secondary 1-5, Junior College 1-2 and Year 1-6).
+* **Academic stream**: Mainstream tracks in Singapore (i.e. Express, NA, NT, IP, IB) as well as other common exam streams (e.g. IELTS, SAT, ACT).
+* **CLI**: Command Line Interface - a type of user interface through which users interact with the app in the form of text inputs only.
+* **GUI**: Graphical User Interface - a type of user interface through which users interact with the app via visual representations.
+* **JAR**: Java Archive - a file format used for aggregating multiple Java class files and their associated components (e.g. images) into a single file for distribution.
+* **Makeup lesson**: A lesson that occurs only once, for a student who has missed a previous lesson.
+* **Mainstream OS**: Windows, Linux, Unix, OS-X.
```

Comment:
Perhaps can include a Lesson Rates to specify that lesson rates refer to cost per hour